cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java

```diff
@@ -163,6 +165,12 @@
    */
   private static final int REPORT_INTERVAL_SEC = 10;
 
+  /**
+   * every "REFRESH_CLIENT_SEC" seconds, a dataClientRefresher thread will try to refresh one thrift
+   * connection for each nodes other than itself.
+   */
+  private static final int REFRESH_CLIENT_SEC = 1;
```

Comment:
Why do you set default value to one? It seems too frequent.

Reply:
And could you please add some descriptions in jira 1488?

Reply:
> And could you please add some descriptions in jira 1488?
added
> Why do you set default value to one? It seems too frequent.
This is an empirical number, and it can be increased. In a recent user scenario, the number of concurrent users may be 40 or 50, and the user perceiving a timeout is about 15 minutes, so if the background thread can check all the clients in a one-minute loop, almost all timeouts can be avoided. This PR is just to support this user's launch on 0.12 as soon as possible. More elaborate policies need to be developed in Issue1487, including but not limited to TTL for all ClientPools.